CRM/Utils/MailjetCustomisationHooks.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
```

Comment:
This was not written by CiviCRM LLC, but by you :) Feel free to skip the license summary, and to add your name / Systopia if you want to.
Can you also rename the file and class to CRM/Mailjet/Utils/Hooks?